### ISSUE TITLE
Cannot load schema from node_modules

### ISSUE THREAD

**Issue by biern** [dotansimha/graphql-code-generator#2322]
**Describe the bug**

Codegen fails to load schema that is located somewhere in `node_modules`. I could successfully load our schema when it was in any other location in the project. If I move it to `node_modules` loading fails with:
```
  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate ./types/gql.ts
      ✖ Load GraphQL schemas
        → Failed to load schema
        Load GraphQL documents
        Generate


 Found 1 error

  ✖ ./types/gql.ts
    Failed to load schema from ../../node_modules/schema.graphql.
```

Exactly the same config / loading path used to work with 1.4.0. 
We keep graphql schema as a separate npm package so loading schema from node modules is an important use case for us.

**To Reproduce**

Try to load a schema from another package in `node_modules` or just move your schema to `node_modules/` directory.

3. My `codegen.yml` config file:

```yml
schema: ../../node_modules/schema.graphql
generates:
  ./types/gql.ts:
    plugins:
      - add: '// tslint:disable'
      - typescript
      - typescript-resolvers
...
```

**Expected behavior**
I can load `schema.graphql` from another package.

**Environment:**

- OS: linux
- `@graphql-codegen/1.5.0`
- NodeJS: 10.16

**Comment by dotansimha** [comment]
Fixed in `graphql-toolkit` :)

**Comment by dotansimha** [comment]
Fixed in 1.6.0.
